apps/daimo-mobile/src/view/screen/InviteScreen.tsx

```diff
@@ -0,0 +1,370 @@
+  return (
+    <View>
+      {showInviteesCount && (
+        <TextCenter>
+          <TextBody color={color.primary}>
+            You've invited {invitees?.length}{" "}
+            {invitees?.length === 1 ? "friend" : "friends"}
+          </TextBody>
+        </TextCenter>
+      )}
+      {showInviteesCount && showUsesLeft && <Spacer h={8} />}
+      {showUsesLeft && (
+        <TextCenter>
+          <TextBody color={showInviteesCount ? color.grayMid : color.primary}>
+            {usesLeft} {usesLeft === 1 ? "invite" : "invites"} left
+          </TextBody>
+        </TextCenter>
+      )}
+    </View>
+  );
+}
+
+const headerTitle = "Invite your friends and earn USDC!";
+
+function LockedHeader() {
+  const goHome = useExitToHome();
+
+  return (
+    <View>
+      <ScreenHeader title="Invite Friends" onExit={goHome} />
+      <HeaderGraphic />
+      <Spacer h={32} />
+      <TextCenter>
+        <TextH2>{headerTitle}</TextH2>
+      </TextCenter>
+    </View>
+  );
+}
+
+function LockedFooter() {
+  const nav = useNav();
+  const sendPaymentLink = () =>
+    nav.navigate("SendTab", {
+      screen: "SendNav",
+      params: { autoFocus: true },
+    });
+
+  return (
+    <View>
+      <Spacer h={32} />
+      <TextCenter>
+        <Octicons name="lock" size={36} color={color.gray3} />
+      </TextCenter>
+      <Spacer h={32} />
+      <TextCenter>
+        <TextBody color={color.gray3}>
+          Use Daimo more to unlock invites. Send a payment link to onboard your
+          contacts.
+        </TextBody>
+      </TextCenter>
+      <Spacer h={32} />
+      <ButtonMed type="primary" title="SEND" onPress={sendPaymentLink} />
+    </View>
+  );
+}
+
+function Header({
+  invitees,
+  inviteCodeStatus,
+}: {
+  invitees: EAccount[];
+  inviteCodeStatus: DaimoInviteCodeStatus;
+}) {
+  const goHome = useExitToHome();
+
+  return (
+    <View>
+      <ScreenHeader title="Invite Friends" onExit={goHome} />
+      <HeaderGraphic invitees={invitees} />
+      <Spacer h={8} />
+      <HeaderCountText
+        invitees={invitees}
+        usesLeft={inviteCodeStatus.usesLeft}
+      />
+      <Spacer h={32} />
+      <TextCenter>
+        <TextH2>{headerTitle}</TextH2>
+      </TextCenter>
+    </View>
+  );
+}
+
+function ReferralButtonsFooter({
+  inviteCodeStatus,
+}: {
+  inviteCodeStatus: DaimoInviteCodeStatus;
+}) {
+  const { link, bonusDollarsInvitee, bonusDollarsInviter } = inviteCodeStatus;
+
+  const onShare = useCallback(async () => {
+    const url = formatDaimoLink(link);
+    shareURL(url);
+  }, [link]);
+
+  const subtitle = (() => {
+    if (
+      bonusDollarsInvitee &&
+      bonusDollarsInviter &&
+      bonusDollarsInvitee === bonusDollarsInviter
+    ) {
+      return `we'll send you both $${bonusDollarsInvitee} USDC`;
+    } else if (bonusDollarsInvitee) {
+      return `we'll send them $${bonusDollarsInvitee} USDC`;
+    } else if (bonusDollarsInviter) {
+      return `we'll send you $${bonusDollarsInviter} USDC`;
```

Comment:
works as long as `bonusDollars...` is always integer, otherwise we'll get eg `$5.5` instead `$5.50`

in practice it will be an integer , looks fine to me

Reply:
if `bonusDollarsInvt[ee,er]` are both 0

then `subtitle` will be undefined

& it'll say just "You'll get credit for the invite on their profile and"

i recommend moving the `and` to the subtitles above ^

+ maybe add a `else return ""` for clarity

Reply:
> works as long as bonusDollars... is always integer, otherwise we'll get eg $5.5 instead $5.50

I think also in copy like this that's a message it's fine / better to omit the full cent.

Good catch on the on the copy bug, fixed as you recommended.